style.css

```diff
@@ -0,0 +1,78 @@
+.main_box {
+    font-size: x-large;
+    height:100% ;
+    width: 300px;
+    position: relative;
+    top: 0;
+    left: 0;
+    padding-top: 20px;
+    background-color: lightblue;
+    display: flex;
+    gap: 20px;
+    
+}
+.fa-ul{
+    margin-right: -20px;
+
+    
+    
+  
+    
+    
+}
+.fa-li{
+    margin-left: -63px;
+}
+
+#you{
+    margin-left: 55px;
+}
+#me{
+    margin-left: 26px;
+}
```

Comment:
I think instead of using an id selector a class attribute must be used

Reply:
- Good job! but please always remember to give reasons for suggesting a change, this will help the author of the project understand better why he needs to make the change. 😄 😄